src/copyediting/logic.py

```diff
@@ -105,36 +110,44 @@ def handle_file_post(request, copyedit):
         return errors
 
 
-def request_author_review(copyedit, article, request):
+def setup_author_review(copyedit, article, request):
```

Comment:
No need for the request to be passed here anymore, in fact, I think we can do away with this function entirely.